Cubical/Categories/Presheaf/Constructions.agda

```diff
@@ -0,0 +1,25 @@
+{-# OPTIONS --safe #-}
+
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Functor
+open import Cubical.Categories.Presheaf.Base
+
+module Cubical.Categories.Presheaf.Constructions where
+
+open Functor
+
+module Single
+  {ℓC ℓ'C ℓSet : Level}
+  {C : Category ℓC ℓ'C}
+  where
+
+  constantPresheaf : hSet ℓSet → Presheaf C ℓSet
```

Comment:
There's already https://github.com/agda/cubical/blob/master/Cubical/Categories/Functors/Constant.agda

Reply:
Oops, didn't think of looking that way, thanks!